src/libs/EmojiUtils.ts

```diff
@@ -199,31 +183,27 @@ function addSpacesToEmojiCategories(emojis) {
 
 /**
  * Get a merged array with frequently used emojis
- * @param {Object[]} emojis
- * @returns {Object[]}
  */
-function mergeEmojisWithFrequentlyUsedEmojis(emojis) {
+function mergeEmojisWithFrequentlyUsedEmojis(emojis: PickerEmojis) {
     if (frequentlyUsedEmojis.length === 0) {
         return addSpacesToEmojiCategories(emojis);
     }
 
-    const mergedEmojis = [Emojis.categoryFrequentlyUsed].concat(frequentlyUsedEmojis, emojis);
+    const mergedEmojis = ([Emojis.categoryFrequentlyUsed] as PickerEmojis).concat(frequentlyUsedEmojis, emojis);
     return addSpacesToEmojiCategories(mergedEmojis);
 }
 
 /**
  * Get the updated frequently used emojis list by usage
- * @param {Object|Object[]} newEmoji
- * @return {Object[]}
  */
-function getFrequentlyUsedEmojis(newEmoji) {
+function getFrequentlyUsedEmojis(newEmoji: Emoji | Emoji[]) {
     let frequentEmojiList = [...frequentlyUsedEmojis];
 
     const maxFrequentEmojiCount = CONST.EMOJI_FREQUENT_ROW_COUNT * CONST.EMOJI_NUM_PER_ROW - 1;
     const currentTimestamp = moment().unix();
-    _.each([].concat(newEmoji), (emoji) => {
+    ([] as Emoji[]).concat(newEmoji).forEach((emoji) => {
```

Comment:
This logic might be inconsistent because `Emoji` and `Emoji[]` are allowed here

```suggestion
[...newEmoji].forEach((emoji) => {
```